client/modules/IDE/components/CopyableInput.jsx

```diff
@@ -42,7 +43,12 @@ class CopyableInput extends React.Component {
           onMouseLeave={this.onMouseLeaveHandler}
         >
           <label className="copyable-input__label" htmlFor={`copyable-input__value-${label}`}>
-            {label}
+            <div style={{ 'display': 'flex', 'justify-content': 'space-between' }}>
```

Comment:
can you add these styles to the scss? all of the scss lives in `client/styles`. you'll need to add a class to this div!